src/components/SingleReviewContainer/SingleReviewContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router"
+
+import Review from "../GenericComponents/Review"
+import ListOfComments from "./ListOfComments"
+
+import * as api from '../../api'
+
+
+const SingleReviewContainer = () => {
+    
+    const [reviews, setReviews] = useState()
+    const [isLoading, setIsLoading] = useState(true)
+
+    const [comments, setComments] = useState()
+    const [isLoadingComments, setIsLoadingComments] = useState(true)
+
+    const { review_id } = useParams()
+
+    useEffect(() => {
+        api.fetchReviewByID(review_id).then((reviews) => {
+            setReviews(reviews)
+            setIsLoading(false)
+        })
+    }, [])
+
+    useEffect(() => {
+        api.fetchCommentsByID(review_id).then((comments) => {
+            setComments(comments)
+            setIsLoadingComments(false)
+        })
+    }, []) //will change empty array once 'add comment form' is added.
+
+    return(
+        <>
+            {isLoading 
+            ? <p>Loading</p> 
+            : reviews.map(review => <Review key={review.review_id} review={review} />)}
+
```

Comment:
I would consider a way of bringing this together to avoid two loading messages being displayed at the same time. As a user, I would expect something like:

- The review is loading, so I get a loading message at the top of the screen
- If the review is loaded but the comments are loading, then I get a loading message underneath the displayed review
- If both are loaded then I just get the normal view


Not a biggie, but something to consider for later